www/prettier.config.mjs

```diff
@@ -0,0 +1,26 @@
+import baseConfig from "../prettier.config.mjs";
+
+/**
+ * @type {import('prettier').Config & import('prettier-plugin-tailwindcss').options &
+ *       import("@ianvs/prettier-plugin-sort-imports").PluginConfig}
+ */
+const config = {
+  ...baseConfig,
+  plugins: [
+    "@ianvs/prettier-plugin-sort-imports",
+    "prettier-plugin-astro",
```

Comment:
is the astro support there yet?

Reply:
yup! https://github.com/withastro/prettier-plugin-astro/releases/tag/v0.11.0
https://github.com/FinnDore/create-t3-app/blob/e5f52c9731d89df768eb67aaa623116c911ff6f6/www/package.json#L57